blaze/compute/tests/test_pandas_compute.py

```diff
@@ -709,6 +709,106 @@ def test_str_lower():
     result = compute(expr, df).reset_index(drop=True)
     assert_series_equal(expected, result)
 
+@pytest.mark.parametrize('old,new,max,expected', (
+    ('o', 'x', None, 'Sxmexne'),
+    ('o', 'x', 1, 'Sxmeone')))
+def test_str_replace(old, new, max, expected):
+    df = pd.Series(['Someone'], name='name')
```

Comment:
Here and below:  could you expand the test data to include a `None` element, to test the logic on missing values?

So instead of `df = pd.Series(['Someone'], name='name')`, we'd have `df = pd.Series(['Someone', None], name='name')`.